tests/conftest.py

```diff
@@ -584,7 +584,12 @@ def pytest_collection_modifyitems(session, config, items):
 
 def pytest_configure(config):
     bins = config.getoption('--redis-server')[:]
-    REDIS_SERVERS[:] = bins or ['/usr/bin/redis-server']
+    cmd = 'which redis-server'
```

Comment:
This code must be executed if `bins` value is empty. Otherwise if there is no `redis-server` in `PATH` but
user has specified `--redis-server` argument this could would throw error.
Please move it under `if` condition:
```python
if not bins:
    os.popen(...)
    ...
```